src/frontend/web_application/src/components/MessageList/components/Message/presenter.jsx

```diff
@@ -146,48 +155,44 @@ class Message extends Component {
                 <Icon type={message.type} className="m-message__type-icon" spaced />
               </div>
             )}
-
-            <Moment className="m-message__date" format="LT" locale={locale}>
-              {message.date}
-            </Moment>
-
-            <DropdownControl toggle={dropdownId} className="m-message__actions-switcher">
-              <Icon type="ellipsis-v" />
-            </DropdownControl>
-
-            <Dropdown
-              id={dropdownId}
-              className="m-message__actions-menu"
-              position="left"
-              closeOnClick
-            >
-              <MessageActionsContainer
-                className="m-message__actions-container"
-                __={__}
-                message={message}
-                onDelete={onDelete}
-              />
-            </Dropdown>
+            {message.date &&
+              <Moment className="m-message__date" format="LT" locale={locale}>
+                {message.date}
+              </Moment> }
+
+            <MessageActionsContainer
+              message={message}
+              dropdownId={this.dropdownId}
+              onDelete={onDelete}
+              onMessageRead={onMessageRead}
+              onMessageUnread={onMessageUnread}
+              __={__}
+            />
 
           </div>
 
-          {subject &&
-            <TextBlock className={subjectClassName}>
-              <Icon type="comments-o" rightSpaced />{subject}
-            </TextBlock>
-          }
+          <div className={subjectClassName}>
```

Comment:
I don't like much this div without condition on subject since a message may not have a subject : for example, a text message (sms) will always render this empty div.